eclipse/plugins/org.switchyard.tools.models.switchyard1_0/src/org/switchyard/tools/models/switchyard1_0/soap/impl/MessageComposerTypeImpl.java

```diff
@@ -187,7 +273,9 @@ public String toString() {
         if (eIsProxy()) return super.toString();
 
         StringBuffer result = new StringBuffer(super.toString());
-        result.append(" (unwrapped: ");
+        result.append(" (copyNamespaces: ");
+        if (copyNamespacesESet) result.append(copyNamespaces); else result.append("<unset>");
```

Comment:
This is all generated code, so I'm not going to mess with it at the risk of screwing something up in EMF, but I think I understand what you're after.